optimum/exporters/onnx/convert.py

```diff
@@ -83,34 +88,43 @@ def validate_encoder_decoder_model_outputs(
             The names of the outputs to check.
         atol (`float`):
             The absolute tolerance in terms of outputs difference between the reference and the exported model.
-        encoder_onnx_model (`Path`):
-            The path to the exported encoder ONNX model.
-        decoder_onnx_model (`Path`):
-            The path to the exported decoder ONNX model.
-        decoder_with_past_onnx_model (`Optional[Path]`, defaults to `None`):
-            The path to the exported decoder with past ONNX model. Required when `past_key_values` are exported.
+        output_dir (`Path`):
+            Output directory where the exported ONNX models are stored.
+        fn_get_models_from_config (`Callable`):
+            Function with signature (`PreTrainedModel`, `exporters.onnx.config.OnnxConfig`) --> `Dict[str, Tuple[Union[PreTrainedModel, TFPreTrainedModel], OnnxConfig]`
```

Comment:
Would put the signature in the type annotation itself, and a description here instead.